index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="iamsrk">
+    <title>Notes using Html & Css</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div  class="paper">
+        <div class="paper-content">
+            <textarea autofocus>Type here...</textarea>
```

Comment:
what is auto foucs ??

Reply:
The autofocus attribute is a boolean attribute.,
 it specifies that the element should automatically get focus when the page loads.